maestro-cli/src/main/java/maestro/cli/graphics/SkiaFrameRenderer.kt

```diff
@@ -0,0 +1,167 @@
+package maestro.cli.graphics
+
+import org.jetbrains.skia.Canvas
+import org.jetbrains.skia.Color
+import org.jetbrains.skia.Font
+import org.jetbrains.skia.FontMgr
+import org.jetbrains.skia.Paint
+import org.jetbrains.skia.Rect
+import org.jetbrains.skia.Surface
+import org.jetbrains.skia.paragraph.FontCollection
+import org.jetbrains.skia.paragraph.ParagraphBuilder
+import org.jetbrains.skia.paragraph.ParagraphStyle
+import org.jetbrains.skia.paragraph.TextStyle
+import org.jetbrains.skiko.toImage
+import java.awt.image.BufferedImage
+import javax.imageio.ImageIO
+
+class SkiaFrameRenderer : FrameRenderer {
```

Comment:
I am thinking about what would be the best way to get this from beta to prod (stable):

1. Recommending to use this one to everyone who raised a Github issue for the remote renderer
2. Announcing this in the contributor's lounge or just the announcement channel to see if it works for everyone.

WDYT?

Reply:
Yeah I think we can announce in slack and have folks try it out. Lot's of users have asked for this